rmvm.rb

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env ruby
+# Copyright 2014, CoverMyMeds, LLC
+# Author: Doug Morris <dmorris@covermymeds.com>/Scott Merrill <smerrill@covermymeds.com>
+# Released under the terms of the GPL version 2
+#   http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt
+require 'rubygems'
+gem 'rbvmomi', '1.6.0'
+require 'optparse'
+require 'rbvmomi'
+require 'yaml'
+require 'io/console'
+
+# establish a couple of sane default values
+options = {
+  'username'   => ENV['USER'],
+  'insecure'   => true,
```

Comment:
Both of these options should use symbols, rather than strings, for the hash keys:

```ruby
options = {
  :username => ENV['USER'],
  :insecure   => true,
}
```

I'd also argue that `insecure` should not be the default.  Let folks set that in their own `.mkvm.yaml` config.